astropy/conftest.py

```diff
@@ -19,6 +19,7 @@
 import pytest
 
 from astropy import __version__
+import astropy
```

Comment:
Why?
```suggestion
```

Reply:
Agreed, this seems extraneous to the proposed changes.

Reply:
Thank you for pointing this out. I meant to remove it (it was for testing purposes).

Reply:
This change has been reverted.